homeassistant/components/fortigate/README.md

```diff
@@ -0,0 +1,49 @@
+# Fortigate Presence Sensor
```

Comment:
This needs to go to the documentation at home-assistant.io

Reply:
https://github.com/home-assistant/home-assistant.io/pull/9807

Reply:
This file is still left. Please remove it.

Reply:
removed, thanks